content/60.methods.md

```diff
@@ -93,17 +93,22 @@ Recording the evolution of information over time and assembling a resource that
 #### Contributor Recruitment and Roles
 
 A preliminary requirement for this undertaking was to establish Manubot as a platform accessible to researchers with limited experience working with git, as is common in biology and medicine, where version control is not typically emphasized [@doi:10.1109/SE4Science.2017.11; @doi:10.1177/2515245918754826; @doi:10.1186/1751-0473-8-7].
-Contributors were recruited by word of mouth and on Twitter.
-We sought out opportunities to integrate existing efforts to train early-career researchers.
+Contributors were recruited primarily by word of mouth and on Twitter, though we also collaborated with existing efforts to train early-career researchers.
 We invited potential collaborators to contribute a short introduction on a GitHub issue in order to collect information about who was involved and provide an introduction to working with GitHub issues.
+
 Interested participants were encouraged to contribute in several ways.
 One option was to catalog articles of interest as issues in the GitHub repository.
 We developed a standardized set of questions for contributors to consider when evaluating an article following a framework often used for assessing medical literature.
 This approach emphasizes examining the methods used, assignment (whether the study was observational or randomized), assessment, results, interpretation, and how well the study extrapolates [@doi:10.5014/ajot.60.4.367].
 Contributors were also invited to contribute or edit text using GitHub's pull request system.
 These contributions were not strictly defined and could range from minor corrections to punctuation and grammar to large-scale additions of text.
+Finally, a small number of contributors (the authors of this paper) contributed technical expertise, either through the development of standardized approaches to the evaluation of papers based on the MAARIE Framework [@isbn:9780781774260], the writing of code to generate manuscript figures, or the addition of features to Manubot.
+All of these additions were also submitted as pull requests, either to the COVID-19 review repository or to an external repository, as appropriate.
+
 Each pull request was reviewed and approved by at least one other contributor before being merged into the main branch.
 We tagged potential reviewers based on the introductions they had contributed in order to encourage participation.
+Authorship was determined based on CRediT [@url:https://casrai.org/credit].
```

Comment:
Reviewer 3 suggested using footnotes for URLs, and I recommend that for URL citations as well.  The URL citations do not translate well to the conference LaTeX template and are quite difficult to fix manually.  I think we could keep URL citations for the small number of URLs that refer to an article with a specific author, title, and date (like `url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers`) and convert the rest.

Pandoc does support footnotes (see https://github.com/manubot/rootstock/issues/352).  We would need to see how they show up in the long HTML document and the converted tex file.

```suggestion
Authorship was determined based on the Contributor Roles Taxonomy [@url:https://casrai.org/credit].
```

Reply:
We could follow up in that rootstock issue to discuss modifying 02.delete-me.md.  There is always a gray area about how much to demonstrate in that example document.  We try to not fully reproduce the pandoc documentation when the feature is pure pandoc but also need to demonstrate common use cases.